### PR TITLE
Li's submission

### PR DESCRIPTION
**Pull request by LZDCA**
Do I need to do this?

### CODE REVIEW

File: LZ/TRY.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+x = "15"
+y = int(x) ** 2
+print('''
+Seanny is going to eat a hamburger
```
Comment (Sadface1000) [inline LZ/TRY.py:4]: seriously?????

File: homework/lesson2/LZDCA-HmWk-2-2.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# HOMEWORK: LESSON 2 : VARIABLES
+# THE CUSTOMER INPUTS ARE SKIPED. THEY ARE ASSIGNED HERE
+
+import statistics
```
Comment (Shuzhengz) [inline homework/lesson2/LZDCA-HmWk-2-2.py:4]: cool, although we haven't gone through libraries yet

**Review by Shuzhengz — APPROVED** [review]
look good

File: homework/lesson2/LZDCA-HmWk-2-2.py (hunk 2 above)
Comment (LZDCA) [reply homework/lesson2/LZDCA-HmWk-2-2.py:4]: I looked for help online. I like python, a lot of people are using it and a lot of help I can get

Comment (Shuzhengz) [reply homework/lesson2/LZDCA-HmWk-2-2.py:4]: yea, looking for help online is one of the most frequent thing software developers do

**Pull request merged**